core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -49,6 +49,14 @@ trait SharedBackendConf {
   def loginConf = sparkConf.getOption(PROP_LOGIN_CONF._1)
   def userName: Option[String] = sparkConf.getOption(PROP_USER_NAME._1)
   def password: Option[String] = sparkConf.getOption(PROP_PASSWORD._1)
+  private[h2o] def credentials: Option[String] = {
```

Comment:
I think we have similar code to this in Rest API Utils, could it be reused?

Reply:
Will take a look, thanks!